packages/block-library/src/blocks/sidecar/scss/_block-limits.scss

```diff
@@ -0,0 +1,13 @@
+.alignwide,
+.align-wide,
+[data-align="wide"] {
+  --block-limit-left: ws;
+  --block-limit-right: we;
+}
+
+.alignfull,
+.align-full,
+[data-align="full"] {
+  --block-limit-left: fs;
+  --block-limit-right: fe;
+}
```

Comment:
Why does this make sense to be in it's own file?

Reply:
At start, we had more things we. Now I totally agree with you that these lines should be moved and this file deleted.